src/marketplace/eqp/v1/packages.md

```diff
@@ -92,7 +92,7 @@ Both `POST` and `PUT` requests support a batch model where multiple packages can
 |platform|string|GET, POST, PUT|technical|yes|The Magento platform compatibility of this package.|`M1`, `M2`|
 |prices|array|GET, POST, PUT|marketing|no|The list of prices in USD set for this package by edition, and the respective installation prices (if any). Editions must match `version_compatibility`.|Array of sub-objects.|
 |prices[N].currency_code|string|GET, POST, PUT|marketing|no|The currency code for this price|Currently only `USD`|
-|prices[N].edition|string|GET, POST, PUT|marketing|no|The magento edition for this price|`CE`, `EE`, `ECE`|
+|prices[N].edition|string|GET, POST, PUT|marketing|no|The Magento edition for this price|`CE`, `EE`, `ECE`|
```

Comment:
I can only comment on changed lines. I see some places in this table that need to be updated.

Delete M1 on line 92 and 122?
On line 120, delete reference to M2 only
Line 123 lists ancient version of M2

Reply:
Changes made. 👍